lib/email/receiver.rb

```diff
@@ -1298,7 +1315,7 @@ def create_post(options = {})
       if result.post
         @incoming_email.update_columns(topic_id: result.post.topic_id, post_id: result.post.id)
         if result.post.topic&.private_message? && !is_bounce?
-          add_other_addresses(result.post, user)
+          add_other_addresses(result.post, user, @mail)
```

Comment:
Do we need to pass @mail?

I found just one reference to that method, so maybe lets just use @mail inside?

Reply:
Ah yes sorry this was a change from the other fix I have to do in this error that snuck into the diff, I will get rid of it for now.